src/uct/ib/rc/base/rc_iface.c

```diff
@@ -584,6 +593,20 @@ UCS_CLASS_INIT_FUNC(uct_rc_iface_t, uct_iface_ops_t *tl_ops,
 #endif
     max_ib_msg_size             = uct_ib_iface_port_attr(&self->super)->max_msg_sz;
 
+    if (dev->flags & UCT_IB_DEVICE_FLAG_ECE) {
+        if (config->ece == UCS_ULUNITS_AUTO) {
+            self->config.ece = 0xffffffffU;
+        } else {
+            self->config.ece = config->ece;
+        }
+    } else if (config->ece == UCS_ULUNITS_AUTO || config->ece == 0) {
+        self->config.ece = 0;
+    } else {
+        ucs_error("ECE operation is not supported");
```

Comment:
ucs_error("%s: cannot set ECE value to 0x%x since the device does not support ECE");

Reply:
Done